src/backend.ts

```diff
@@ -36,7 +36,9 @@ export interface SaveFunc<M> {
 }
 
 export interface Process<M> {
-  (node: Instance<M>, path: string, liveOnly: boolean): Promise<ProcessResult>;
+  (node: Instance<M>, path: string, liveOnly: boolean): Promise<
+    Partial<ProcessResult>
```

Comment:
Had to add `Partial` to please typescript. It was complaining about line 162 that the given default keys where always being overridden, since this is not true and our process responses can be partial this needed to be added.